src/wasm/flb_wasm.c

```diff
@@ -121,6 +121,10 @@ struct flb_wasm *flb_wasm_instantiate(struct flb_config *config, const char *was
     size_t dir_index = 0;
 #endif
 
+#ifdef FLB_WASI_SOCKETS
+    const char **default_address_pool = {"127.0.0.1"};
```

Comment:
Compiler generates warning in this line for pointer type glitches. How should we handle this?

Reply:
I changed the declaration to get rid of the warning.